core/const.go

```diff
@@ -23,6 +23,9 @@ const (
 	EdgeXService     = "edgex"
 	VaultToken       = "X-Vault-Token"
 
+	// Edgex Microservices
+	Coredata = "core"
+
```

Comment:
If this is a username, please update the comment so it's clear.

Reply:
This may not be needed as we are using the mongoinit as the single path. Will remove this const.